### PR TITLE
Major Readme changes 

### PR DESCRIPTION
**Pull request by T4shf33n**
Important changes for readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
-# unc0ver
-### The most advanced jailbreak tool
+# Unc0ver
+### The most advance jailbreak tool
```
Comment (alexslee) [inline README.md]: I think that should still say `advanced`

File: README.md [hunk 2]
```diff
@@ -113,3 +118,5 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * [@Swag_iOS](https://twitter.com/Swag_iOS) for testing
 * [@jailbreakbuster](https://twitter.com/jailbreakbuster) for testing
```
Comment (king4q) [inline README.md]: That’s not my Twitter handler. It’s @King4q1

**Review by endercypher — CHANGES_REQUESTED** [review]

**Comment by king4q** [comment]
@pwn20wndstuff @sbingner

File: README.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
-# unc0ver
-### The most advanced jailbreak tool
+# Unc0ver
+### The most advance jailbreak tool
 ![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
 
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
```
Comment (thehappydinoa) [inline README.md]: Should update after the release of iOS 12 jailbreak to update.

File: README.md [hunk 4]
```diff
@@ -52,10 +55,11 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * Perform a full restore with Rollectra before switching from the other jailbreaks
 * Turn on the AirPlane Mode before starting the jailbreak
 * Turn off Siri before starting the jailbreak
+* Wait atleast one minute on the jailbreak screen
```
Comment (thehappydinoa) [inline README.md]: > at least

**Review by thehappydinoa — COMMENTED** [review]
Could I suggest reordering the screenshots as it occurs on launch?

File: README.md (hunk 3 above)
Comment (T4shf33n) [reply README.md]: We’ll see that

File: README.md (hunk 4 above)
Comment (T4shf33n) [reply README.md]: Done

**Comment by T4shf33n** [comment]
@thehappydinoa  Screen shots are on the way

File: README.md (hunk 2 above)
Comment (T4shf33n) [reply README.md]: Updated and sorry

**Comment by thehappydinoa** [comment]
Would you like me to send screenshots from iPhone instead of iPad?

**Comment by T4shf33n** [comment]
@thehappydinoa  I don’t have a iPhone if you have one please send me

**Comment by thehappydinoa** [comment]
main_screen
settings

**Comment by Chasewhip8** [comment]
I can give you iPhone X screenshots

**Comment by T4shf33n** [comment]
@Chasewhip8 please forward I will choose carefully 👌

**Comment by T4shf33n** [comment]
@thehappydinoa You forgot the lower part...eg - restart , diagnostics

**Comment by Chasewhip8** [comment]
We should wait on screenshots for the iOS 12 update. Build number will change and exploit selection will as well

File: README.md [hunk 5]
```diff
@@ -39,6 +39,7 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * Button to restart device
 * Button to open Cydia in case it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Buitl-In Update checker
```
Comment (thehappydinoa) [inline README.md]: *Built-In

Comment (T4shf33n) [reply README.md]: I see... I am editing on iPad so doing a lot of mistakes.. sorry 😇

Comment (T4shf33n) [reply README.md]: Resolved..thanks

**Comment by T4shf33n** [comment]
@Chasewhip8 we will update then also.. Pwn is busy in jailbreak so we should do the rest

File: README.md [hunk 6]
```diff
@@ -39,12 +39,19 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
 * Button to open Cydia in case it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
-* Buitl-In Update checker
+* Built-In Update checker
 * Working debugserver
 * Detailed debugging 
 * Realtime Log window 
 * An awesome UI
 
+## Currently supported exploits
```
Comment (thehappydinoa) [inline README.md]: Maybe we could add some iOS versions for each exploit

File: README.md [hunk 7]
```diff
@@ -66,32 +78,15 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
 * Enable the on-fly entitlement patching on iOS 11: Work in progress
 * WebKit Port with [@_niklasb](https://twitter.com/_niklasb)'s [WebKit Exploit](https://github.com/phoenhex/files/tree/master/exploits/ios-11.3.1): Work in progress
 
 ## Screenshots
-<img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-1.PNG?token=AlyO4wXUInR6oHEgx0Tg31ri0t1q91frks5bx5ZbwA%3D%3D" width="281.25" height="609" /> <img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-2.PNG?token=AlyO48vs-YYcaKUgxXh8nIEUQQz_QEoqks5bx5ZqwA%3D%3D" width="281.25" height="609" /> <img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-3.PNG?token=AlyO44tYr5-jl7Pg0jup0tCqm3rSjUhiks5bx5Z4wA%3D%3D" width="281.25" height="609" />
+<img src="https://github.com/T4shf33n/Undecimus/blob/master/Resources/IMG_0332.PNG" width="281.25" height="609" /> <img src="https://github.com/T4shf33n/Undecimus/blob/master/Resources/IMG_0333.PNG" width="281.25" height="609" /> <img src="https://github.com/T4shf33n/Undecimus/blob/master/Resources/IMG_0334.PNG" width="281.25" height="609" />
```
Comment (thehappydinoa) [inline README.md]: Can we change the paths to something like `/Resources/screenshot_1.png`? Also, can we clean up file names?

File: README.md [hunk 8]
```diff
@@ -1,31 +1,31 @@
...
 
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
 by [@pwn20wnd](https://twitter.com/Pwn20wnd) & [@sbingner](https://twitter.com/sbingner)<br/>
 UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](https://reddit.com/u/Samg_is_a_Ninja)<br/>
 
 ## The most outstanding changes over the other jailbreaks
 * All exploits in same app
-* Detailed error messages
+* Detailed and specific error messages
```
Comment (thehappydinoa) [inline README.md]: This is redundant

File: README.md [hunk 9]
```diff
@@ -1,31 +1,31 @@
-# unc0ver
+# Unc0ver
 ### The most advanced jailbreak tool
 ![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
```
Comment (thehappydinoa) [inline README.md]: Can we try to fix all of these weird links?

**Review by thehappydinoa — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (T4shf33n) [reply README.md]: Can you suggest anything ?

File: README.md (hunk 9 above)
Comment (T4shf33n) [reply README.md]: Then I will have to Re-upload uncover logo

Comment (thehappydinoa) [reply README.md]: I don't think so, we can just change it to `unc0ver logo`

**Comment by T4shf33n** [comment]
I would look at the rest tomorrow

**Comment by T4shf33n** [comment]
@thehappydinoa  Relative path is not working on Screenshot 1

**Comment by thehappydinoa** [comment]
Made a PR

**Comment by T4shf33n** [comment]
@thehappydinoa done ✅

File: README.md (hunk 8 above)
Comment (thehappydinoa) [reply README.md]: I would just remove it

File: README.md [hunk 10]
```diff
@@ -1,31 +1,31 @@
...
 ## The most outstanding changes over the other jailbreaks
 * All exploits in same app
-* Detailed error messages
+* Detailed and specific error messages
 * Faster patches
 * More stable patches
 * No extra battery drain
 * No random freezes
-* No random slow downs
+* No random slow downs/lags
```
Comment (thehappydinoa) [inline README.md]: I this is also redundant.

File: README.md [hunk 11]
```diff
@@ -1,31 +1,31 @@
...
 * No data is logged or shared
-* No malware
+* No malware/spyware
```
Comment (thehappydinoa) [inline README.md]: I this is also redundant.

File: README.md [hunk 12]
```diff
@@ -1,31 +1,31 @@
...
 * Telesphoreo port for ARM64
-* Much faster Cydia
+* Much faster Cydia interaction
 * Much more stable Cydia
-* Much more modern looking and acting Cydia
+* Much more modern user interface based Cydia
 * Cydia skips uicache when not needed
 * Cydia supports iPhone X screen size
-* Cydia Substrate for tweak injection
+* Official Cydia Substrate for tweak injection
```
Comment (thehappydinoa) [inline README.md]: I am not sure its official...

File: README.md [hunk 13]
```diff
@@ -35,12 +35,23 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
 * Button to open Cydia in case it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Built-In Update checker
 * Working debugserver
+* Detailed debugging 
+* Realtime Log window 
 * An awesome UI
 
+## Currently supported exploits
+* VFS(Empty_list) by lan beer
```
Comment (thehappydinoa) [inline README.md]: iOS 11-11.3.1

File: README.md [hunk 14]
```diff
@@ -35,12 +35,23 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
+* MultiPath by lan beer
```
Comment (thehappydinoa) [inline README.md]: iOS 11.2-11.3

File: README.md [hunk 15]
```diff
@@ -35,12 +35,23 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
+* Async_Wake by lan beer
```
Comment (thehappydinoa) [inline README.md]: iOS 11.1.2

File: README.md [hunk 16]
```diff
@@ -35,12 +35,23 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
+* Vouchar Swap by Brandon Azad
```
Comment (thehappydinoa) [inline README.md]: Typo: `Voucher Swap`

iOS 11.2-12.1.2

File: README.md [hunk 17]
```diff
@@ -35,12 +35,23 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
+* V1ntex by Tihmstar
```
Comment (thehappydinoa) [inline README.md]: iOS 11.2-11.4.1

**Review by thehappydinoa — CHANGES_REQUESTED** [review]

File: README.md (hunk 12 above)
Comment (T4shf33n) [reply README.md]: Yes. It is

Comment (thehappydinoa) [reply README.md]: Ah ok

File: README.md (hunk 14 above)
Comment (T4shf33n) [reply README.md]: MultiPath supports 1.3.1

**Comment by T4shf33n** [comment]
@thehappydinoa Updated and fixed

**Review by thehappydinoa — APPROVED** [review]
LGTM

**Comment by T4shf33n** [comment]
Anything new to add ? Like a new section or useful links?

**Comment by thehappydinoa** [comment]
I don't think so, I think this looks really good

**Comment by T4shf33n** [comment]
@thehappydinoa ok so mentioning them to notify ✌️

@pwn20wndstuff @Cryptiiiic @sbingner

File: README.md (hunk 1 above)
Comment (haykam821) [reply README.md]: Yeah, 'most advance' is improper grammar.

**Comment by haykam821** [comment]
Screenshots should be from an unjailbroken iPhone (no status bar tweaks), have full cellular service (no Wi-Fi), and stay at 9:41. You should try to emulate Apple's screenshot style as much as possible.

**Comment by Chasewhip8** [comment]
Ye I can do that on release

**Comment by T4shf33n** [comment]
@Chasewhip8 @haykam821  the problem is 1) I do not use SIM card on iPad 

2) I’ll have to change system date if to do so cause by then I go out

**Comment by Chasewhip8** [comment]
I’ll give them to you once it has been released

**Comment by T4shf33n** [comment]
@Chasewhip8 Looking forward to it

**Comment by Chasewhip8** [comment]
@T4shf33n check out [this](https://github.com/T4shf33n/Undecimus/pull/2) pull request.

**Comment by T4shf33n** [comment]
@Chasewhip8 merged your request 

@thehappydinoa would you please review it?

File: README.md (hunk 1 above)
Comment (T4shf33n) [reply README.md]: It’s a typo 🙂

**Comment by Chasewhip8** [comment]
Will need to change iOS 11 to iOS 11 and 12 for the cydua part eventually

File: README.md [hunk 18]
```diff
@@ -1,46 +1,53 @@
...
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
 by [@pwn20wnd](https://twitter.com/Pwn20wnd) & [@sbingner](https://twitter.com/sbingner)<br/>
 UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](https://reddit.com/u/Samg_is_a_Ninja)<br/>
 
 ## The most outstanding changes over the other jailbreaks
-* All exploits in same app
-* Detailed error messages
+* All exploits in the same app
+* Detailed  error messages
```
Comment (haykam821) [inline README.md]: Was it necessary to add an extra space here?

File: README.md [hunk 19]
```diff
@@ -1,46 +1,53 @@
...
 * No data is logged or shared
 * No malware
 * Proper jailbreak state detection
 * Proper bootstrap extraction to fix issues such as Cydia not appearing after jailbreak
-* Native build of Cydia for iOS 11
+* Native and updated build of Cydia for iOS 11
 * Telesphoreo port for ARM64
-* Much faster Cydia
-* Much more stable Cydia
-* Much more modern looking and acting Cydia
```
Comment (haykam821) [inline README.md]: Instead of removing these three points, why not add them to the other mention of Cydia in the feature list?

File: README.md [hunk 20]
```diff
@@ -1,46 +1,53 @@
...
 * Cydia skips uicache when not needed
 * Cydia supports iPhone X screen size
-* Cydia Substrate for tweak injection
-* Much faster ldrestart 
-* Much more stable ldrestart
-* Changes to Cydia were made with permission from Saurik 
+* Official Cydia Substrate for tweak injection
+* Much faster and more stable ldrestart
```
Comment (haykam821) [inline README.md]: I'd put this in an inline code block.

File: README.md [hunk 21]
```diff
@@ -1,46 +1,53 @@
...
-* Much more modern looking and acting Cydia
 * Cydia skips uicache when not needed
 * Cydia supports iPhone X screen size
-* Cydia Substrate for tweak injection
-* Much faster ldrestart 
-* Much more stable ldrestart
-* Changes to Cydia were made with permission from Saurik 
+* Official Cydia Substrate for tweak injection
+* Much faster and more stable ldrestart 
+* Changes to Cydia made with permission from Saurik
```
Comment (haykam821) [inline README.md]: Like previously suggested with other (removed) items, you should add this to the first mention of Cydia in the list.

File: README.md [hunk 22]
```diff
@@ -1,46 +1,53 @@
...
+* Changes to Cydia made with permission from Saurik 
 * Option to skip loading daemons
 * Option to dump APTicket
 * Option to refresh icon cache
 * Option to disable auto updates
 * Option to block app revokes
 * Option to restore RootFS
-* Button to restart device
-* Button to open Cydia in case it doesn't appear on the Home Screen
+* Option to Export tfp0
```
Comment (haykam821) [inline README.md]: Don't capitalize `Export`.

File: README.md [hunk 23]
```diff
@@ -1,46 +1,53 @@
...
 * Working debugserver
+* Detailed debugging 
+* Realtime Log window 
 * An awesome UI
 
+## Currently supported exploits
+* VFS(Empty_list) by lan beer (11-11.3.1)
+* MultiPath by lan beer (11.2-11.3.1)
+* Async_Wake by lan beer (11.1.2)
+* Voucher Swap by Brandon Azad (11.2-12.1.2)
```
Comment (haykam821) [inline README.md]: Use the proper underscored names and their exploit identifier if possible.

File: README.md [hunk 24]
```diff
@@ -1,46 +1,53 @@
...
+* Button to restart the device
+* Button to open Cydia if it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Built-In Update checker
 * Working debugserver
+* Detailed debugging 
+* Realtime Log window 
 * An awesome UI
 
+## Currently supported exploits
```
Comment (haykam821) [inline README.md]: Add a space before starting the list.

Comment (haykam821) [inline README.md]: Add a comment about the exploit support. How about:

> Multiple exploits across multiple iOS versions are supported.

File: README.md [hunk 25]
```diff
@@ -52,47 +59,30 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
+- [ ] Chain [@_bazad](https://twitter.com/_bazad)'s [blanket](https://github.com/bazad/blanket) to bypass the developer certificate requirement for multi_path
+- [ ] Enable the on-fly entitlement patching on iOS 11
+- [ ] WebKit Port with [@_niklasb](https://twitter.com/_niklasb)'s [WebKit Exploit](https://github.com/phoenhex/files/tree/master/exploits/ios-11.3.1)
 
 ## Screenshots
-<img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-1.PNG?token=AlyO4wXUInR6oHEgx0Tg31ri0t1q91frks5bx5ZbwA%3D%3D" width="281.25" height="609" /> <img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-2.PNG?token=AlyO48vs-YYcaKUgxXh8nIEUQQz_QEoqks5bx5ZqwA%3D%3D" width="281.25" height="609" /> <img src="https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Resources/Screenshot-3.PNG?token=AlyO44tYr5-jl7Pg0jup0tCqm3rSjUhiks5bx5Z4wA%3D%3D" width="281.25" height="609" />
+<img src="/Resources/s1.PNG" width="281.25" height="609" /> <img src="/Resources/s2.PNG" width="281.25" height="609" /> <img src="/Resources/s3.PNG" width="281.25" height="609" />
```
Comment (haykam821) [inline README.md]: For cleaner code, put each image on its own line.

File: README.md [hunk 26]
```diff
@@ -52,47 +59,30 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
-* ~~v1.1.1: Add a label to show the ECID and a button to open the source code, improve auto layout and fix various bugs in RootFS remount, RootFS restore, RootFS resource copier, Icon cache refresher, Version checker, Exploit selector, jailbreak state detector and others~~
-* ~~v1.1.2: Improve auto layout and code and Significantly improve Empty_List (VFS) exploit and slightly improve Multi_Path (MPTCP)~~
-* ~~v1.1.3: Fix a bug in starting jailbreakd~~
-* ~~v1.1.4: Fix a bug in finding offsets:  [Download (IPA)](https://github.com/pwn20wndstuff/Undecimus/raw/master/Resources/Undecimus.ipa)~~
-* Releases are now available at https://github.com/pwn20wndstuff/Undecimus/releases
+* Releases are now available [here](https://github.com/pwn20wndstuff/Undecimus/releases)
```
Comment (haykam821) [inline README.md]: It won't be 'now available' in the future.

File: README.md [hunk 27]
```diff
@@ -52,47 +59,30 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
...
 
 ## Special Thanks
```
Comment (haykam821) [inline README.md]: Add a link to the GitHub repo contributor page at the end of this list.

File: README.md [hunk 28]
```diff
@@ -103,7 +93,7 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * [@theninjaprawn](https://twitter.com/theninjaprawn) for the patchfinder64 additions
 * [@saurik](https://twitter.com/saurik) for Cydia and Substrate
 * [@FCE365](https://twitter.com/FCE365) for the empty_list reliability improvements
-* Credits for [Undecimus-Resources](https://github.com/pwn20wndstuff/Undecimus-Resources)
+#### Credits for [Undecimus-Resources](https://github.com/pwn20wndstuff/Undecimus-Resources)
```
Comment (haykam821) [inline README.md]: Link to the repo contributor page for that repo.

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Comment by T4shf33n** [comment]
@haykam821 create a PR in the fork to make thlngs clear

File: README.md (hunk 19 above)
Comment (ZaWarud0) [reply README.md]: What about "Native build of Cydia for iOS 11 that is faster and much more stable." or something?

File: README.md [hunk 29]
```diff
@@ -1,46 +1,52 @@
...
 * Option to disable auto updates
 * Option to block app revokes
 * Option to restore RootFS
-* Button to restart device
-* Button to open Cydia in case it doesn't appear on the Home Screen
+* Option to Export tfp0
+* Button to restart the device
+* Button to open Cydia if it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Built-In Update checker
```
Comment (ZaWarud0) [inline README.md]: Built-in update checker

No need for extra capitalization.

File: README.md [hunk 30]
```diff
@@ -1,46 +1,52 @@
...
+* Realtime Log window 
 * An awesome UI
-* Substrate and Cydia Support 
+
+## Currently supported exploits
+* VFS(Empty_list) by lan beer (11-11.3.1)
+* MultiPath by lan beer (11.2-11.3.1)
+* Async_Wake by lan beer (11.1.2)
+* Voucher Swap by Brandon Azad (11.2-12.1.2)
+* V1ntex by Tihmstar (11.2-11.4.1)
```
Comment (ZaWarud0) [inline README.md]: * V3ntex by Tihmstar (12.0-12.1.2) partial support.

File: README.md (hunk 26 above)
Comment (ZaWarud0) [reply README.md]: Yeah change it to 
* Releases are available [here](https://github.com/pwn20wndstuff/Undecimus/releases)

File: README.md (hunk 30 above)
Comment (T4shf33n) [reply README.md]: will be added after full release or ETA sons will be hyped

File: README.md (hunk 18 above)
Comment (T4shf33n) [reply README.md]: fixed

File: README.md (hunk 19 above)
Comment (T4shf33n) [reply README.md]: ''Native build of Cydia for iOS 11 that is faster and much more stable'' adapted the line

File: README.md (hunk 20 above)
Comment (T4shf33n) [reply README.md]: An example would be helpful

File: README.md (hunk 21 above)
Comment (T4shf33n) [reply README.md]: done

File: README.md (hunk 22 above)
Comment (T4shf33n) [reply README.md]: resolved

File: README.md (hunk 23 above)
Comment (T4shf33n) [reply README.md]: Didn't undertand it properly

File: README.md (hunk 24 above)
Comment (T4shf33n) [reply README.md]: that would be in-specific

File: README.md (hunk 29 above)
Comment (T4shf33n) [reply README.md]: done

File: README.md (hunk 28 above)
Comment (T4shf33n) [reply README.md]: done

File: README.md (hunk 25 above)
Comment (T4shf33n) [reply README.md]: That would Re-locate the images

File: README.md (hunk 20 above)
Comment (haykam821) [reply README.md]:
```suggestion
* Much faster and more stable `ldrestart`
```

File: README.md (hunk 24 above)
Comment (haykam821) [reply README.md]: Exactly. Put the unspecific sentence first, then list all the supported exploits to clarify.

**Comment by clarityzzz** [comment]
I'm not against the idea in this but please be more careful : screenshots look way off : they're stretched and you can't discern anything on them.

**Comment by T4shf33n** [comment]
@clarityzzz I will just re-size the images.. thanks for the point

**Comment by Chasewhip8** [comment]
Once the ph luv build drops or I get s private build I can do the screenshots for this.

**Comment by clarityzzz** [comment]
Screenshots still look wayyy off on desktop, too stretched. Are they from an iPad? I'd suggest from an iPhone X.
I also suggest adding an example form on how to post a new issue : way too many issues without proper information or diagnostics.

**Comment by Chasewhip8** [comment]
Yes I have an iPhone X and will do it once I have the new build it s public release. Or pwn or someone else could do them.

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: README.md [hunk 31]
```diff
@@ -1,46 +1,57 @@
...
 * Label to show the days left till the application expires
+* Built-in Update checker
 * Working debugserver
+* Detailed debugging 
+* Realtime Log window 
 * An awesome UI
 
+## Currently supported exploits
+
+# Multiple exploits supported across multiple iOS version
```
[the remark this answers is not in the document]
Comment (T4shf33n) [reply README.md]: Thanks but I just merged a PR by whip regarding this issue

File: README.md [hunk 32]
```diff
@@ -52,47 +63,30 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * Perform a full restore with Rollectra before switching from the other jailbreaks
 * Turn on the AirPlane Mode before starting the jailbreak
 * Turn off Siri before starting the jailbreak
+* Wait at least one minute on the jailbreak screen
 
 ## Source code
-* This project is completely open source and it will be kept like it in the future
-* Any kind of contribution is welcome
-* The source code can be found on [pwn20wndstuff](https://github.com/pwn20wndstuff)'s GitHub account
```
[the remark this answers is not in the document]
Comment (T4shf33n) [reply README.md]: Re-added

**Comment by Chasewhip8** [comment]
@T4shf33n check pr’s

**Comment by Chasewhip8** [comment]
PR is [here](https://github.com/T4shf33n/Undecimus/pull/4)

Other than that @pwn20wndstuff @Samgisaninja what do you think?

**Comment by Samgisaninja** [comment]
The screenshots need to be resized or something, idk what device they’re taken from though. 

I think we will probably wait until everything is merged from iOS 12. Also every single PR I’ve merged so far has messed something up in some way, so I’ll let pwn handle that if he wants to.

**Comment by Chasewhip8** [comment]
Hmm maybe just grab the readme in the end as that should be the only file changed. And for the device screenshots we were waiting to take them on an updated unc0ver but I’m guessing the UI isn’t going to change from what I heard in discord. Do you want us to update them now or after ?

**Comment by Chasewhip8** [comment]
Ye, it seems like some apt/ files are in this PR so maybe exclude them on the merge

**Comment by Samgisaninja** [comment]
The UI will probably be tweaked slightly (exploitpicker branch), I’ll probably re-visit this a week or so after iOS 12 support is added

**Comment by Chasewhip8** [comment]
Sounds good

**Comment by haykam821** [comment]
You should make a separate pull request for iOS 12-related tweaks to the README.

**Comment by Chasewhip8** [comment]
Not really as this is going to be merged after iOS 12 jb is released so might as well do it in here

**Comment by haykam821** [comment]
@Chasewhip8 Unless you rebase, you shouldn't do that.

**Comment by Chasewhip8** [comment]
Ye I’m not the owner of this pr. I would fix the merge problems if I was.

**Comment by T4shf33n** [comment]
@chasewhip8 @hykam821 I am also waiting for the unc0ver update and for someone to send the screenshots from iphone and also if you have a change then leave  a review . I'll look into it and sorry for the delay as I was flooded with my work.

**Comment by T4shf33n** [comment]
@Chasewhip8  The apt changes were made when you pulled from the master branch and made a PR remember?

**Comment by Chasewhip8** [comment]
My bad that would do it haha

**Comment by Chasewhip8** [comment]
I'm having issues reverting that commit. Maybe you can try?

**Comment by T4shf33n** [comment]
@Chasewhip8  The files have been updated by Pwn since that merge so GitHub is showing an error. Now ?

**Comment by T4shf33n** [comment]
I see two options 1) Re-opening another Pull request copying the changes from this one 2) Pulling again from the latest master branch before merging this.

I prefer Option - 1 . What about you guys?

**Comment by Chasewhip8** [comment]
Just rebase it

**Comment by Chasewhip8** [comment]
Ye it is showing me an error as well

**Comment by Chasewhip8** [comment]
Actually let me try something

**Comment by haykam821** [comment]
> I see two options 1) Re-opening another Pull request copying the changes from this on

Please don't ever do that.

**Comment by T4shf33n** [comment]
@haykam821 i was just exploring options but chase had made a PR in my fork to fix this . Help us to review the fix

**Comment by Chasewhip8** [comment]
My fix will work as long as when they pull dev to master they should make dev priority
Need to fix some other things in it fn

**Comment by Chasewhip8** [comment]
Ok, recheck the pr. This should work if they merge with develop priority.

**Comment by T4shf33n** [comment]
@Chasewhip8 You have done it congrats 😸

**Comment by prakchanraksmey** [comment]
Need to fix

**Comment by Chasewhip8** [comment]
@prakchanraksmey what?

**Pull request closed without merging**